src/main/java/frc/lib/constants/VisionConstants.java

```diff
@@ -22,7 +22,7 @@
 public class VisionConstants {
   // AprilTag layout
   public static AprilTagFieldLayout aprilTagLayout =
-      AprilTagFieldLayout.loadField(AprilTagFields.kDefaultField);
+      AprilTagFieldLayout.loadField(AprilTagFields.k2025ReefscapeAndyMark);
```

Comment:
Is this just for the field at WPI, or is this something we need to configure at game time?